client/sites-dashboard/components/sites-dashboard-ctas.tsx

```diff
@@ -1,18 +1,22 @@
 import { useI18n } from '@wordpress/react-i18n';
 import { useSelector } from 'react-redux';
+import { useAddNewSiteUrl } from 'calypso/lib/paths/use-add-new-site-url';
 import getCurrentQueryArguments from 'calypso/state/selectors/get-current-query-arguments';
-import { useSitesDashboardCreateSiteUrl } from '../hooks/use-sites-dashboard-create-site-url';
+import { AppState } from 'calypso/types';
 import { useSitesDashboardImportSiteUrl } from '../hooks/use-sites-dashboard-import-site-url';
+import { TRACK_SOURCE_NAME } from '../utils';
 import { EmptyStateCTA } from './empty-state-cta';
 
 export const CreateSiteCTA = () => {
 	const { __ } = useI18n();
 
 	const isHostingFlow = useSelector(
-		( state ) => getCurrentQueryArguments( state )?.[ 'hosting-flow' ] === 'true'
+		( state: AppState ) => getCurrentQueryArguments( state )?.[ 'hosting-flow' ] === 'true'
```

Comment:
Instead if this, you can import `useSelector` from `calypso/state` (it'll automatically have the type info! more info: p4TIVU-azW-p2)

Reply:
Ah, this is nice! Finally, we have that :)